tests/test_identity_v2.py

```
@@ -382,6 +382,84 @@ def test_collect_result_actions_compatibility_and_drift_v2(self):
         action_tuple = replacements[file_rel_path][6][0]
         self.assertEqual(action_tuple[2], "你好世界")
 
+    def test_adapter_plan_preserves_batch_non_chinese_policy_allowance(self):
+        file_rel_path = "screens_patronlistitem.rpy"
+        file_path = os.path.join(batch_mod.legacy.TL_DIR, file_rel_path)
+        lines = ['    "Alpha, Beta, Gamma"\n']
+        Path(file_path).write_text("".join(lines), encoding="utf-8")
+        scanned = runtime.scan_all_translation_units(lines, file_rel_path)
+        item_id, (line, start, end, source_text) = next(iter(scanned.items()))
+        translated_text = "Alpha， Beta， Gamma"
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)